sdk/core/core-rest-pipeline/src/policies/bearerTokenChallengeAuthenticationPolicy.ts

```diff
@@ -136,7 +136,9 @@ export function bearerTokenChallengeAuthenticationPolicy(
   // The options are left out of the public API until there's demand to configure this.
   // Remember to extend `BearerTokenChallengeAuthenticationPolicyOptions` with `TokenCyclerOptions`
   // in order to pass through the `options` object.
-  const cycler = createTokenCycler(credential /* , options */);
+  const getAccessToken = credential
+    ? createTokenCycler(credential /* , options */).getToken
+    : (_scopes: string | string[], _options: GetTokenOptions) => Promise.resolve(null);
```

Comment:
can't this just be

```suggestion
    : () => Promise.resolve(null);
```

Or does TS complain?

Reply:
Probably not. I was trying to align with the signature of the other case, but it is unnecessary.